### DISCUSSION TITLE
C152/C172 mapping questions

### DISCUSSION THREAD

**Discussion by fmichea** [nguyenquyhy/Flight-Tracker-StreamDeck#273] [Q&A]
Hello,

More of a set of questions than a feature request. I was trying to map the C152/C172 (G1000) in MSFS 1.16.2.0 with latest plugin version and hit the following issues. Goal was to be able to do checklist for everything, C152 was quite easy, but I only got to "After Starting Engine" page on C172. Most likely stuff I'm looking for exists, just can't quite figure it out. Hoping for some hints. Here are my questions:

Is there a way to get the current MAGNETO status? I can see the events necessary to control it in TOGGLE_EVENT.cs but cannot find a corresponding value in TOGGLE_VALUE.cs to fetch the current status.

![image](https://user-images.githubusercontent.com/953489/121972295-ff6d7600-cd7a-11eb-91b2-5acb2fd3b522.png)

Part of the starting procedure of the C172 is to switch and fiddle with these switches. TOGGLE_MASTER_BATTERY event seems to affect both STBY BATT (top toggle) and MASTER BAT (right red). Avionics Master switch does both grey. I did not find any way to split wither into two events. Also did not find the event for test function on standby battery. Any of those exist?

I was also trying to use the default profile for COM/MAV, but need an additional button on it to change profile to my main MSFS profile page. I was able to trick SD to make it writeable, but the numpad selection does not work anymore doing that (it asks to install profile). Any options?

Just checking the next few pages, I believe I might also be missing a way to set BARO, didn't see anything else. Would appreciate any help, thank you!

**Comment by ToniZGit** [comment]
Hi, I am in the same situation as you. Have you managed to check the status of the magneto? Thanks.
